cookbooks/tomography_based_plate_motions/plugins/tomography_based_plate_motions.cc

```diff
@@ -1350,6 +1528,23 @@ namespace aspect
                              "We can evaluate the effects of lithospheric thickness variations on the surface plate "
                              "motions using this. Currently, this parameter sets the lithosphere to a constant "
                              "thickness of 100 km.");
+          prm.declare_entry ("Use slab2 database", "false",
+                             Patterns::Bool (),
+                             "This parameter value determines if we want to the slab depths and thickness defined "
+                             "in Slab2 database (Hayes et al., 2018). Currently, it only supports input as a 2D   "
+                             "grid on the boundary with columns representing depths and thickness of the slabs.");
```

Comment:
This second part is not quite clear. Shouldn't that be in the parameter that let's you specify the file location? That's where you would define the format of the file, right?

Reply:
Yes, this sentence is not necessary. I will remove it from the prm_entry.